site/assets/css/fish_style.css

```diff
@@ -263,9 +263,7 @@ h3 {
     image-rendering: -o-crisp-edges;
     image-rendering: -webkit-optimize-contrast;
     image-rendering: -webkit-crisp-edges;
-    image-rendering: pixelated;
-    image-rendering: optimize-contrast; 
-    image-rendering: crisp-edges;
+    image-rendering: optimize-contrast;
```

Comment:
The prior behavior used 'pixelated' only if a browser didn't support `optimize-contrast` or (better) `crisp-edges` - this change now will force nearest-neighbor filtering for most browsers. The duplicate definitions will not overwrite the earlier ones on user agents where the new definition has an unsupported value for the property.